packages/forma-36-website/src/content/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: 'Getting started'
+---
+
+## Installation
+
+### Install components
+
+Inside your React project directory, install Forma 36 by running either of the following:
+
+**When using NPM:**
+
+```bash static=true
+npm i @contentful/f36-components
+```
+
+**When using Yarn:**
+
+```bash static=true
+yarn add @contentful/f36-components`
+```
+
+#### Installing components as separate packages
+
+You can install Forma 36 React components as separate packages if that's what you need for your project. For example, here's how to install just the button:
```

Comment:
[Suggestion for merge]

You can install components as separate packages. Tree shaking will make your bundle contain only the components you actually need. For example, here's how to install the button: